2025-08-13 16:34:15 | INFO     | __main__             | _load_config         | Configuration loaded from config.yaml
2025-08-13 16:34:15 | INFO     | __main__             | _init_connection_pool | Connection pool initialized successfully
2025-08-13 16:34:15 | INFO     | __main__             | __init__             | CSV source directory: D:\NOPPALIT\History\Saturn_Acc_Logs
2025-08-13 16:34:15 | INFO     | __main__             | __init__             | Export directory: D:\NOPPALIT\History\Saturn_Acc_Logs
2025-08-13 16:34:15 | INFO     | __main__             | init_db              | Database initialized successfully
2025-08-13 16:34:15 | INFO     | __main__             | start                | Starting CSV processing service every 1 minutes...
2025-08-13 16:34:15 | INFO     | __main__             | start                | CSV source directory: D:\NOPPALIT\History\Saturn_Acc_Logs
2025-08-13 16:34:15 | INFO     | __main__             | start                | Delete after export: True
2025-08-13 16:34:15 | ERROR    | __main__             | get_connection       | Database operation failed: tuple index out of range
2025-08-13 16:34:15 | ERROR    | __main__             | _import_rows_to_db   | Failed to import rows to database: tuple index out of range
2025-08-13 16:34:15 | ERROR    | __main__             | _process_single_csv  | Failed to process CSV file D:\NOPPALIT\History\Saturn_Acc_Logs\RPB1_20250813_162459.csv: tuple index out of range
2025-08-13 16:34:15 | ERROR    | __main__             | process_all_csv_files | Failed to process RPB1_20250813_162459.csv: tuple index out of range
2025-08-13 16:35:17 | ERROR    | __main__             | get_connection       | Database operation failed: tuple index out of range
2025-08-13 16:35:17 | ERROR    | __main__             | _import_rows_to_db   | Failed to import rows to database: tuple index out of range
2025-08-13 16:35:17 | ERROR    | __main__             | _process_single_csv  | Failed to process CSV file D:\NOPPALIT\History\Saturn_Acc_Logs\RPB1_20250813_162459.csv: tuple index out of range
2025-08-13 16:35:17 | ERROR    | __main__             | process_all_csv_files | Failed to process RPB1_20250813_162459.csv: tuple index out of range
2025-08-13 16:35:18 | INFO     | __main__             | start                | Service interrupted by user
2025-08-13 16:35:18 | INFO     | __main__             | close                | Connection pool closed
2025-08-13 16:35:18 | INFO     | __main__             | cleanup              | Service cleanup completed
